Shared/Extensions/URL-Extensions.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  URL-Extensions.swift
+//  NetNewsWire
+//
+//  Created by Stuart Breckenridge on 03/05/2020.
+//  Copyright © 2020 Ranchero Software. All rights reserved.
+//
+
+import Foundation
+
+extension URL {
+	
+	/// Extracts email address from a `URL` with a `mailto` scheme, otherwise `nil`.
+	var emailAddress: String? {
+		scheme == "mailto" ? URLComponents(url: self, resolvingAgainstBaseURL: false)?.path : nil
+	}
+	
+	/// Extracts telephone number from a `URL` with a `tel` scheme, otherwise `nil`.
+	var telNumber: String? {
```

Comment:
TIL! Nice extension to extract email address and tel number 👍 

Also looks like we are not really using the `telNumber` anywhere

Reply:
Removed in `f58ac4d`